fabric-biome-api-v1/src/testmod/java/net/fabricmc/fabric/test/biome/FabricBiomeTest.java

```diff
@@ -63,5 +66,17 @@ public void onInitialize() {
 		OverworldBiomes.addHillsBiome(BiomeKeys.BAMBOO_JUNGLE, BiomeKeys.BASALT_DELTAS, 0.9);
 
 		OverworldBiomes.addContinentalBiome(BiomeKeys.END_HIGHLANDS, OverworldClimate.DRY, 0.5);
+
+		BiomeModifications.create(new Identifier("fabric:test_mod"))
```

Comment:
I would expand the tests a bit, I would atleast add an ore or something as well.

Reply:
Please see the unit tests I linked in the original PR, I don't want to duplicate those in the test mod until it's clear whether they'll go through or not.

Reply:
Those are awesome, they would be really great to have in. They required a loader change didnt they?

Reply:
The commit contains more a "proof of concept" for how to do a loader-change, by placing a class in the loader-package to get around the package-visible accessibility of some loader classes. It obviously can't stay like that. Maybe we can convince player that it _is_ worth supporting, then I'd try to make a loader PR (it shouldn't really be _too_ hard, essentially we'd need a third KnotLauncher that doesnt invoke any entrypoints and uses more or less a dummy GameProvider, which is what that proof of concept does).

Reply:
I do still think there should be few more examples/tests here, atleast add an ore or something.